ros1_bridge/__init__.py

```diff
@@ -213,13 +274,21 @@ def __eq__(self, other):
     def __hash__(self):
         return hash('%s/%s' % (self.package_name, self.message_name))
 
+    def __str__(self):
+        return self.prefix_path + ":" + self.package_name + ":" + self.message_name
+
+    def __repr__(self):
+        return self.__str__()
+
 
 class MappingRule(object):
     __slots__ = [
         'ros1_package_name',
         'ros1_message_name',
+        'ros1_service_name',
         'ros2_package_name',
         'ros2_message_name',
+        'ros2_service_name',
```

Comment:
I changed conditionals so it is either message mapping or service mapping, never both at the same time.